plugins/org.jkiss.dbeaver.ext.ocient/src/org/jkiss/dbeaver/ext/ocient/model/plan/OcientExecutionPlan.java

```diff
@@ -65,7 +65,7 @@ public String getQueryString() {
     @Override
     public String getPlanQueryString() {
         // Adds "explain json" to front of query with proper spacing
-        return String.format("explain json %s", query);
+        return "explain json" + " " + query;
```

Comment:
Please, just a single space:

```suggestion
        return "explain json " + query;
```

It's really obvious that the space is essential here, no need to make it even more obvious and smelly.